src/components/selection_input/selection_input.ts

```diff
@@ -47,6 +50,8 @@ interface Props {
   onSelectionRemoved?: (index: number) => void;
   onSelectionConfirmed?: () => void;
   colors?: Color[];
+  disabledRanges?: boolean[];
+  disabledTitle?: string;
```

Comment:
```suggestion
  disabledRangeTitle?: string;
```
nitpick. Probably a bit better ?